src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -162,6 +163,7 @@ private void addBanUsernameMenu(final JMenu parentMenu) {
       try {
         controller.banUsername(new Node(name1, InetAddress.getByName("0.0.0.0"), 0), new Date(expire));
       } catch (final UnknownHostException ex) {
+        ClientLogger.logQuietly(ex);
```

Comment:
I'm pretty sure looking up "0.0.0.0" will only fail if there are absolutely no network interfaces installed (not even loopback).  So it's unlikely to ever fail, but I figured logging it can't hurt in the rare case someone attempts to run the lobby on a machine with no networking support.

Please advise if the log should be removed and simply replaced with a comment.

(This comment applies to all newly-added log calls in this file.)